src/app.js

```diff
@@ -8,8 +8,13 @@ import renderTasks from "./todo";
 import * as myDom from "./dom";
 import clearElement from "./utilis";
 
-let lists =
-  JSON.parse(localStorage.getItem(myDom.LOCAL_STORAGE_LIST_KEY)) || [];
+let lists = JSON.parse(localStorage.getItem(myDom.LOCAL_STORAGE_LIST_KEY)) || [
```

Comment:
- Do not disable linter rules. Fix all errors. This was already pointed out in the previous reviews.